.gitignore

```diff
@@ -56,3 +56,4 @@ pip-wheel-metadata/
 
 # pytest debug logs generated via --debug
 pytestdebug.log
+__pycache__/
```

Comment:
nitpick (feel free to ignore): This should maybe be in the big block above, not under "# pytest debug logs generated via --debug"